Makefile

```diff
@@ -83,7 +87,7 @@ test-scripts:
 
 # Format code with Black
 format:
-	$(RUN) black scripts/ tests/ modules/Clinical/tests/ modules/WES/tests/
+	$(RUN) black scripts/ tests/ modules/Clinical/tests/ modules/WES/tests/ modules/*/src/
```

Comment:
I'd separate each module on a new line to make it cleaner to add/remove in future